src/tests/SelectedRevisionsTable.test.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from 'react';
+
+import SelectedRevisionsTable from '../components/Search/SelectedRevisionsTable/SelectedRevisionsTable';
+import { render, store, screen } from './utils/test-utils';
+
+describe('Search View', () => {
+  let mockStore: any;
+  let mockDispatch: any;
+
+  beforeEach(() => {
+    mockStore = jest.spyOn(store, 'getState');
+    mockDispatch = jest.spyOn(store, 'dispatch');
+  });
+
+  it('renders correctly when there are no revisions', () => {
+    mockStore.mockReturnValue({
+      search: {},
+      compareTable: {
+        revisions: [],
+      },
+    } as any);
+
+    render(<SelectedRevisionsTable />);
+    expect(screen.getByText('Add some revisions')).toBeInTheDocument();
+    expect(document.body).toMatchSnapshot();
+  });
+
+  it('should render correctly when there are revision and delete accordingly', async () => {
+    mockStore.mockReturnValue({
+      search: {},
+      compareTable: {
+        revisions: [
+          {
+            id: 0,
+            revision: 'coconut',
+            author: 'john@python.com',
+            push_timestamp: 42,
+            repository_id: 4,
+            revisions: [
+              {
+                revision: 'coconut',
+                author: 'john@python.com',
+                comments: 'This is my first message',
+              },
+            ],
+          },
+        ],
+      },
+    } as any);
+
+    render(<SelectedRevisionsTable />);
+    expect(screen.getByText('Maximum 4 revisions.')).toBeInTheDocument();
+
+    // testing delete functionality
+    const button = document.querySelector('#close-button') as HTMLButtonElement;
+
+    await button.click();
+
+    expect(mockDispatch).toHaveBeenCalled();
```

Comment:
I'm not sure if this is specific enough. Can you add a check that the deleted item is no longer in the document? example: https://github.com/mozilla/perfcompare/blob/master/src/tests/SearchView.test.tsx#L142-L144